Competition/src/test/java/competition/subsystems/gripper/ReleaseDiscCommandTest.java

```diff
@@ -0,0 +1,34 @@
+package competition.subsystems.gripper;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import competition.BaseCompetitionTest;
+import competition.subsystems.gripper.commands.GrabDiscCommand;
+
+public class ReleaseDiscCommandTest extends BaseCompetitionTest {
+
+    GrabDiscCommand grabDisc;
+    GripperSubsystem gripperSubsystem;
+    @Override
+    public void setUp() {
+        super.setUp();
+        grabDisc = this.injector.getInstance(GrabDiscCommand.class);
+        gripperSubsystem = this.injector.getInstance(GripperSubsystem.class);
+    }
+
+    @Test
+    public void testGrabDisc() {
+        gripperSubsystem.gripperPiston.setOn(true);
+        assertTrue("grabHatchet is true", gripperSubsystem.gripperPiston.getAdjusted());
+
+        gripperSubsystem.releaseHatch();
```

Comment:
You need to use the Command, not the Subsystem (see my other comments on the other command test)